tests/NHSD.BuyingCatalogue.Ordering.Services.UnitTests/NHSD.BuyingCatalogue.Ordering.Services.UnitTests.csproj

```diff
@@ -0,0 +1,26 @@
+<Project Sdk="Microsoft.NET.Sdk">
+
+  <PropertyGroup>
+    <TargetFramework>net5.0</TargetFramework>
+
+    <IsPackable>false</IsPackable>
+  </PropertyGroup>
+
+  <ItemGroup>
+    <PackageReference Include="AutoFixture.NUnit3" Version="4.15.0" />
+    <PackageReference Include="FluentAssertions" Version="5.10.3" />
+    <PackageReference Include="NUnit" Version="3.13.1" />
+    <PackageReference Include="NUnit3TestAdapter" Version="3.17.0" />
+    <PackageReference Include="Microsoft.NET.Test.Sdk" Version="16.9.1" />
+  </ItemGroup>
```

Comment:
Also please make sure to install the latest pre-release version of StyleCop.

```powershell
Install-Package StyleCop.Analyzers -Version 1.2.0-beta.333
```

Reply:
Installed package `StyleCop.Analyzers`